### ISSUE TITLE
questions about VoiceBank+DEMAND and STFT

### ISSUE THREAD

**Issue by miccio-dk** [yxlu-0102/MP-SENet#75]
Dear authors,
thank you for sharing this code and for the great article!
I have a couple of questions:
* in the training script, you're using the test set for validation and to select the best checkpoint; shouldn't you use a separate validation set (perhaps a subset of the training data with unseen speakers) for that, so as to avoid overfitting?
* in both the paper and the code you mention nfft=400, hop=100 (i.e., 75% overlap), and window=Hann; however, when using the Hann window, COLA is only guaranteed for 50% overlap, so shouldn't this use a squared Hann window?

Thank you!!
